packages/catalog-process/test/db.test.ts

```diff
@@ -1877,7 +1877,8 @@ describe("database test", async () => {
         const descriptor2: Descriptor = {
           ...mockDescriptor,
           id: generateId(),
-          state: descriptorState.draft,
```

Comment:
To avoid breaking the previous tests, I decided to not use draft for those tests. I added specific `draft`-related tests below.